File: probabilidad-estadistica/practica-1.py
from random import randint

total_experimentos = 1000000
contador_evento = 0

examen = [randint(1, 3) for i in range(10)]

for i in range(total_experimentos):
    calificacion = 0

    for respuesta in examen:
        if (randint(1, 3) == respuesta):
            calificacion = calificacion + 1

    if calificacion == 6:
        contador_evento = contador_evento + 1

print(contador_evento / total_experimentos)

File: probabilidad-estadistica/practica-1.lua
local total_experimentos = 1000000
local contador_eventos = 0
local examen = {}

for i = 1, 10 do
  examen[i] = math.random(3)
end

for i = 1, total_experimentos do
  local calificacion = 0
  for j = 1, #examen do
    if examen[j] == math.random(3) then
      calificacion = calificacion + 1
    end
  end
  if calificacion == 6 then
    contador_eventos = contador_eventos + 1
  end
end

print(contador_eventos / total_experimentos)
